cmd/skaffold/app/cmd/build_test.go

```diff
@@ -26,47 +26,73 @@ import (
 
 	"github.com/GoogleContainerTools/skaffold/cmd/skaffold/app/flags"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/build"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
 	"github.com/GoogleContainerTools/skaffold/testutil"
 )
 
 func TestQuietFlag(t *testing.T) {
-	mockCreateRunner := func(context.Context, io.Writer) ([]build.Artifact, error) {
-		return []build.Artifact{{
-			ImageName: "gcr.io/skaffold/example",
-			Tag:       "test",
+	singleArtifactRunner := func(context.Context, io.Writer) ([]build.Result, error) {
+		return []build.Result{{
+			Target: latest.Artifact{
+				ImageName: "gcr.io/skaffold/example",
+			},
+			Result: build.Artifact{
+				ImageName: "gcr.io/skaffold/example",
+				Tag:       "gcr.io/skaffold/example:test",
+			},
 		}}, nil
 	}
 
-	defer func(f func(context.Context, io.Writer) ([]build.Artifact, error)) {
-		createRunnerAndBuildFunc = f
-	}(createRunnerAndBuildFunc)
+	doubleArtifactRunner := func(context.Context, io.Writer) ([]build.Result, error) {
+		return []build.Result{
+			{
+				Target: latest.Artifact{
+					ImageName: "gcr.io/skaffold/image1",
+				},
+				Result: build.Artifact{
+					ImageName: "gcr.io/skaffold/image1",
+					Tag:       "gcr.io/skaffold/image1:tag",
+				},
+			},
+			{
+				Target: latest.Artifact{
+					ImageName: "gcr.io/skaffold/image2",
+				},
+				Error: errors.New("build error"),
+			},
+		}, nil
+	}
+
+	originalBuildFormatFlag := buildFormatFlag
 
 	var tests = []struct {
 		description    string
 		template       string
 		expectedOutput []byte
-		mock           func(context.Context, io.Writer) ([]build.Artifact, error)
+		mock           func(context.Context, io.Writer) ([]build.Result, error)
 		shouldErr      bool
 	}{
 		{
-			description:    "quiet flag print build images with no template",
```

Comment:
Can you add a test case where 
- there is an error  for singleArtifactRunner
- there is an error for one of the doubleArtifactRunner.